sdk/src/main/java/io/customer/sdk/data/model/CustomerIOParsedPushPayload.kt

```diff
@@ -0,0 +1,16 @@
+package io.customer.sdk.data.model
+
+/**
+ * Data class that holds payload for notifications from CIO dashboard
+ *
+ * @property deepLink url received in notification data
+ * @property cioDeliveryId Customer.io message delivery id
+ * @property title notification content title text
+ * @property body notification content body text
+ */
+data class CustomerIOParsedPushPayload(
+    val deepLink: String?,
+    val cioDeliveryId: String,
+    val title: String,
+    val body: String
```

Comment:
I recommend adding the raw push payload's `Bundle` to this object since a customer might be adding custom `data` properties to the JSON push payload sent through CIO.